src/styles/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  width: $calendar-width;
+  padding: $day-padding;
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    background-color: $default-bg;
+    border: $day-border solid $border-color;
+    justify-content: center;
+    align-items: center;
+    width: $day-size;
+    height: $day-size;
+    transition-duration: $day-transform-duration;
+    font-size: $day-font-size;
+    font-family: $day-font-family;
+
+    &::before {
+      content: '';
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: $hover-bg;
+
+      transform: translateY(-20px);
+    }
+  }
+
+  @each $day, $i in $days {
+    &--start-day-#{$day} &__day:first-child {
+      margin-left: ($day-gap + $day-size) * $i;
+    }
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), when using the `@each` directive, the variables should be enclosed in parentheses to improve readability. Please wrap `($day, $i)` in parentheses.